compiler_opt/tools/regalloc_trace/group_functions_lib.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A library that contains utilities for grouping functions."""
+
+import dataclasses
+import os
+import math
+import subprocess
+import json
+
+from compiler_opt.rl import corpus
+
+
+@dataclasses.dataclass
+class FunctionPathAndSize:
+  path: str
+  size: int
+
+
+def _get_functions_chunked_by_command_line(
+    function_folder: str, delete_flags: tuple[str, ...] = ()
+) -> dict[tuple[str], list[str]]:
+  function_corpus = corpus.Corpus(
+      data_path=function_folder,
+      delete_flags=delete_flags,
+      construct_cmd_for_compilation=False)
```

Comment:
used where?

Reply:
Not sure what you're commenting on here. `function_corpus` is used shortly below. We need `construct_cmd_for_compilation` to be set to `False` to avoid getting an extra `-x ir {}` stuck in the command line from the corpus tooling.